exercises/concept/pacman-rules/test/Tests.hs

```diff
@@ -0,0 +1,44 @@
+import PacmanRules (eatsGhost, loses, scores, wins)
+import Test.Hspec (describe, hspec, it, shouldBe)
+
+main :: IO ()
+main = hspec $ do
+  describe "eatsGhost" $ do
+    it "ghost eaten when touched while powered" $ do
+      (eatsGhost True True) `shouldBe` True
+
+    it "ghost not eaten when not touched" $ do
+      (eatsGhost True False) `shouldBe` False
+
+    it "ghost not eaten when touched un-powered" $ do
+      (eatsGhost False True) `shouldBe` False
+
```

Comment:
The tests are not exhaustive, but could be. I'm not sure about the likelihood of false positives, but in any case exhaustivity seems cheap.

Reply:
I added the rest of the cases.